ui-components/src/navigation/Tabs.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FunctionComponent } from "react"
+import "./Tabs.scss"
+
+import {
+  Tab as ReactTab,
+  Tabs as ReactTabs,
+  TabList as ReactTabList,
+  TabPanel as ReactTabPanel,
+} from "react-tabs"
+
+export interface TabsProps {
+  children: React.ReactNode
+  className?: string
+  defaultFocus?: boolean
+  defaultIndex?: number
+  disabledTabClassName?: string
+  domRef?: (node?: HTMLElement) => void
+  forceRenderTabPanel?: boolean
+  onSelect?: (index: number, last: number, event: Event) => boolean | void
+  selectedIndex?: number
+  selectedTabClassName?: string
+  selectedTabPanelClassName?: string
+}
+
+export const Tabs: FunctionComponent<TabsProps> = ({ children, ...props }: TabsProps) => {
+  const className = ["tabs"]
+  if (props.className) className.push(props.className)
+  return (
+    <ReactTabs {...props} className={className.join(" ")}>
+      {children}
+    </ReactTabs>
+  )
+}
+
+export interface TabProps {
+  children: React.ReactNode
+  className?: string
+  disabled?: boolean
+  disabledClassName?: string
+  selectedClassName?: string
+  tabIndex?: string
+}
+
+export const Tab: FunctionComponent<TabProps> = ({ children, ...props }: TabProps) => {
+  const className = ["tabs__tab"]
+  if (props.className) className.push(props.className)
+  return (
+    <ReactTab {...props} className={className}>
+      {children}
+    </ReactTab>
+  )
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
What is this doing / why the ignore?

Reply:
Since I'm directly wrapping the React Tabs components, I have to set the `tabsRole` property on some of the components as described in docs here: https://github.com/reactjs/react-tabs#custom-components

But that line wasn't getting along with TS, not sure why.